controllers/BookmarkController.ts

```diff
@@ -49,9 +51,28 @@ export default class BookmarkController implements BookmarkControllerI {
      * @param {Response} res Represents response to client, including the
      * body formatted as JSON arrays containing the tuit objects that were bookmarked
      */
-    findAllTuitsBookmarkedByUser = (req: Request, res: Response) =>
-        BookmarkController.bookmarkDao.findAllTuitsBookmarkedByUser(req.params.uid)
-            .then((bookmarks: Bookmark[]) => res.json(bookmarks));
+    findAllTuitsBookmarkedByUser = (req: Request, res: Response) => {
+        const uid = req.params.uid;
+        // @ts-ignore
+        const profile = req.session['profile'];
+        const userId = uid === 'me' && profile ?
+            profile._id : uid;
+        if (userId === 'me') {
+            res.sendStatus(403);
+        } else {
+            try {
+                BookmarkController.bookmarkDao.findAllTuitsBookmarkedByUser(userId)
```

Comment:
Hey Andrew, we need a minor change here that we need populate "postedBy" for these tuits.